pkg/generated/restapi/operations/signing_cert.go

```diff
@@ -61,7 +61,7 @@ type SigningCert struct {
 func (o *SigningCert) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 	route, rCtx, _ := o.Context.RouteInfo(r)
 	if rCtx != nil {
-		r = rCtx
+		*r = *rCtx
```

Comment:
do you know why these got converted to pointers?

Reply:
this comes from the commit https://github.com/go-swagger/go-swagger/pull/2553/commits/43f6f80154a4259054764f809b83a624e9c7b171 

since I think they're trying to actually update the value at `r`, they do a shallow copy of the values into the actual memory VS just updating the local copy of the pointer.